roles/sap_ha_prepare_pacemaker/README.md

```diff
@@ -0,0 +1,136 @@
+# sap_ha_prepare_pacemaker Ansible Role
+
+The role **sap_ha_prepare_pacemaker** is necessary because tasks needs to be finished on all nodes before the the cluster can be configured.
+These tasks
+
+*   Software Installation
+*   Host authentication
+
+are part of this role and excluded from the role **sap_ha_install_pacemaker**
+
+## Scope
+
+- **RedHat Enterprise Linux**
+    - Tested on RHEL 8.2, 8.4 and later
+    - Please also check 
+    - [Overview of the Red Hat Enterprise Linux for SAP Solutions subscription](https://access.redhat.com/solutions/3082481)
+    - [Automating SAP HANA Scale-Up System Replication using the RHEL HA Add-On](https://access.redhat.com/articles/3004101)
+
+- **Azure**
+    - Tested
+    - Followed the steps based on the guide published in
+        - [Azure Pacemaker Setup Guide](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-rhel-pacemaker)
+
+- **AWS**
```

Comment:
Would suggest defining the IaaS type, e.g. `, EC2 Instance`

Also to control scope, add the `AWS Classic environment` is out of scope?